### PR TITLE
translation update

### PR DESCRIPTION
**Pull request by sunhang36**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d8369880175b1aebc34a3cdb0d9a81d2e40eb34a/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/da97e39436c3e997b16fb9cb63cf52c04cb1d9c8/index.html)

File: docs/_data/ja/toc_text.yml [hunk 1]
```diff
@@ -1,41 +1,63 @@
...
+# For documentation, see https://developer.shotgunsoftware.com/tk-doc-generator/authoring/toc/
 #
 
+reference: リファレンス
+guides: ガイド
+trusted-solutions: 信頼できるソリューション
+
 rv: RV
 toolkit: Toolkit
-shotgun: ShotGrid
```
Comment (willis102) [inline docs/_data/ja/toc_text.yml:8]: This PR reverts many instances of rebranding such as this...

**Review by willis102 — APPROVED** [review]
I've approved this PR, since I think it should be *technically* fine, however it does revert many instances of rebranding that exist on main.  Do we want to merge this and revert the rebranding now?

  I will approve this but caution that we should consider the rebranding goals before merging.

**Comment by sunhang36** [comment]
Hi @willis102 , I saw there are some conflicts and since rebranding is reverted in this pr, could we wait for next tranlsation drop which should come in the week of June 21th. Rebranding will come with that loop of translations.

**Pull request closed without merging**